<div class="container ">
    <div class="card-body shadow-lg">
        <%= form_with(model: reference ,url: crearReferencia_path) do |form| %>
          <div class="form-group row mb-3">
            <% if reference.errors.any? %>
              <div style="color: red">
                <h2><%= pluralize(reference.errors.count, "error") %> No se puede continuar por los siguientes errores:</h2>

                <ul>
                  <% reference.errors.each do |error| %>
                    <li><%= error.full_message %></li>
                  <% end %>
                </ul>
              </div>
            <% end %>
          </div>

          <div class="form-group row mb-3" style="display:none">
            <%= form.label 'Proveedor ', class:'col-sm-2 ' %>
            <div class="col-sm-4">
                <%= form.text_field :proveedor_id ,class:"form-control" ,:value =>  @id %>
            </div>
          </div>
          
          <div class="form-group row mb-3">
            <%= form.label 'Nombre_entidad:', class:'col-sm-2 ' %>
            <div class="col-sm-4">
            <%= form.text_field :nombre_entidad , class:'form-control' %>

            </div>
          </div>

          <div class="form-group row mb-3">
            <%= form.label 'Nombre_contacto:', class:'col-sm-2 ' %>
            <div class="col-sm-4">
            <%= form.text_field :nombre_contacto , class:'form-control' %>

            </div>
          </div>

          <div class="form-group row mb-3">
            <%= form.label 'Telefono:', class:'col-sm-2 ' %>
            <div class="col-sm-4">
            <%= form.text_field :telefono_contacto , class:'form-control' %>

            </div>
          </div>

          

          

          <button type="submit" name="button" class='btn btn-primary'>
            <span>Guardar</span>
          </button>

          <%= yield(:boton) %>
        <% end %>
      </div>
 </div>